storage/src/main/java/org/apache/uniffle/storage/handler/impl/HdfsShuffleReadHandler.java

```diff
@@ -94,8 +94,8 @@ protected ShuffleIndexResult readShuffleIndex() {
       return new ShuffleIndexResult(indexData, dateFileLen);
     } catch (Exception e) {
       LOG.info("Fail to read index files {}.index", filePrefix, e);
+      throw e;
```

Comment:
Why do we need this? If this is not necessary for this pr, I prefer less change.

Reply:
Done